src/datachain/query/dataset.py

```diff
@@ -296,15 +296,23 @@ def q(*columns):
 
 @frozen
 class Subtract(DatasetDiffOperation):
-    on: Sequence[str]
+    on: Sequence[Union[str, tuple[str, str]]]
```

Comment:
Shouldn't this be `Sequence[tuple[str, str]]`?

Reply:
Yeah.I will update it.Thank you.

Reply:
> Shouldn't this be `Sequence[tuple[str, str]]`?

Yeah.I will update it.Thank you.